### PR TITLE
Allow global use

### PR DESCRIPTION
**Pull request by anteriovieira**
```shell
stats show graham-campbell 
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,28 +19,28 @@ Packagist Stats was created by, and is maintained by [Graham Campbell](https://g
 
 This project is relatively small, and it's usage is pretty simple.
 
-Install dependencies:
+Installation via composer:
 
 ```bash
-$ composer install --no-dev
+$ composer global require graham-campbell/packagist-stats dev-master
```
Comment (GrahamCampbell) [inline README.md]: not dev-master please

Comment (anteriovieira) [inline README.md]: @GrahamCampbell 
The package is not available in a stable-enough version according to your minimum-stability setting.
How do you suggest?

Comment (GrahamCampbell) [inline README.md]: Composer will resolve a version to use. Even then, you should NEVER need dev-master.

`^0.1.*@alpha` or `^0.2.*@dev` will both work fine.

**Comment by anteriovieira** [comment]
OK, I got it.
I will seek to find out more about these specifications.
Thank you.

**Comment by GrahamCampbell** [comment]
I'll probably release v1.0.0 soon anyway. I'd kind of forgotten about this repo tbh. ;)

**Comment by anteriovieira** [comment]
Very good, I liked the idea and I will be supporting whenever possible. :)

**Comment by GrahamCampbell** [comment]
v1.0.0 tagged. :)

**Comment by anteriovieira** [comment]
:+1:  perfect

**Pull request merged**